CRM/Contact/BAO/Query.php

```diff
@@ -6309,15 +6309,13 @@ public static function getQillValue($daoName, string $name, $value, $op, string
   public static function getWildCardedValue($wildcard, $op, $value) {
     if ($wildcard && $op === 'LIKE') {
       if (CRM_Core_Config::singleton()->includeWildCardInName && (substr($value, 0, 1) != '%')) {
-        return "%$value%";
+        $value = "%$value";
```

Comment:
Before, it wrapped the string with `%`s on both sides. After, it only prepends `%`.  

My memory of this codepath is cold -- but wouldn't that be a substantive change in the search behavior?

Reply:
See the following two lines (6314-6315) which add the trailing '%' if it does not already exist.